Vision/Vision.bs

```diff
@@ -114,7 +114,9 @@ Markup Shorthands: markdown yes
 
 	These core values will be clearly demonstrated by how W3C leads the Web forward: 
 	by being inclusive, principled, and continually striving to make the Web better 
-	through these principles and the Ethical Web Principles. 
+	through these principles and the Ethical Web Principles.
```

Comment:
I think it's upside down to refer to more detailed documents here; this one should stand alone

Reply:
this is a separate issue and unrelated to this change (captured in https://github.com/w3c/AB-public/issues/81).